Dockerfile.mysql

```diff
@@ -0,0 +1,12 @@
+FROM mysql:latest
+
+ENV MYSQL_DATABASE=app_db
+ENV MYSQL_ROOT_PASSWORD=1234
+ENV MYSQL_USER=app_user
+ENV MYSQL_PASSWORD=1234
+
+EXPOSE 3306
+
+VOLUME /var/lib/mysql
+
+COPY init.sql /docker-entrypoint-initdb.d/
```

Comment:
![image](https://github.com/user-attachments/assets/62a1b200-d3f5-4703-8d9e-848c796d088b)
Add Eof(End-of-file), need to add a blank line at the end